lib/chef/resource/lwrp_base.rb

```diff
@@ -89,7 +89,7 @@ def self.default_action(action_name=NULL_ARG)
           else
             action = action_name.to_sym
             @actions.push(action) unless @actions.include?(action)
-            @default_action = action
+            @default_action = [action]
```

Comment:
If they set `default_action [:add, :create]` then won't this be a nested array?  You can get around that with `@default_action = [action].flatten`.  You should also add a test case for this scenario.